docs/content/en/docs/tasks/cluster/cluster-scale/baremetal-scale.md

```diff
@@ -54,17 +54,37 @@ If you don't have any available hardware that match this requirement in the clus
 
 #### Upgrade Cluster Command for Scale Up/Down
 
-##### With Hardware CSV File
-
-```bash
-eksctl anywhere upgrade cluster -f cluster.yaml --hardware-csv <hardware.csv>
-```
-
-##### Without Hardware CSV File
-
-```bash
-eksctl anywhere upgrade cluster -f cluster.yaml
-```
+1. eksctl CLI: To create a workload cluster with eksctl, run:
+
+    ##### With Hardware CSV File
+    
+    ```bash
+    eksctl anywhere upgrade cluster -f cluster.yaml --hardware-csv <hardware.csv>
+    ```
+    
+    ##### Without Hardware CSV File
+    
+    ```bash
+    eksctl anywhere upgrade cluster -f cluster.yaml
+    ```
+
+2. kubectl CLI: The cluster lifecycle feature lets you use kubectl to talks to the Kubernetes API to create a workload cluster. To use kubectl, run:
+     ```bash
+     kubectl apply -f eksa-w01-cluster.yaml --kubeconfig mgmt/mgmt-eks-a-cluster.kubeconfig
+     ```
+    As noted earlier, adding the `--kubeconfig` option tells `eksctl` to use the management cluster identified by that kubeconfig file to create a different workload cluster.
+
+3. GitOps**: See [Manage separate workload clusters with GitOps]({{< relref "../../tasks/cluster/cluster-flux.md#manage-separate-workload-clusters-using-gitops" >}})
+
+4. Terraform**: See [Manage separate workload clusters with Terraform]({{< relref "../../tasks/cluster/cluster-terraform.md#manage-separate-workload-clusters-using-terraform" >}})
+
+   >**NOTE**:For kubectl, GitOps and Terraform:
+   >
+   > You can't add additional hardware when scaling workload cluster.
```

Comment:
```suggestion
   > * You can't add additional hardware when scaling workload cluster.
```
If these are two separate issues, perhaps you could add a bullet to each. If you do this, check to make sure it doesn't mess up the formatting.